osf/management/commands/reindex_with_current_metrics_mappings.py

```diff
@@ -0,0 +1,80 @@
+"""
+Reindex data to use current mapping for ES metrics classes
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from elasticsearch_dsl import connections
+from elasticsearch_metrics.registry import registry
+
+logger = logging.getLogger(__name__)
+
+
+def get_metric_class(index_name: str) -> type:
+    app_label, model_name = index_name.split('_')[:2]
+    return registry.all_metrics[app_label][model_name]
+
+
+def increment_index_versions(old_indices: list):
+    """
+    Increment versions numbers for new indices, these kind don't matter because they should always be aliased to
+    the original format of {app_label}_{cls.__name__.lower()}_{year}.
+
+    :param old_indices: indices to be updated
+    :return: indices names that are going to be reindexed into.
+    """
+    new_indices = []
+    for index in old_indices:
```

Comment:
May need some more validation in here; it's currently trusting input to use the actual index name, rather than alias name. This probably needs to get an ES connection to inspect the indices.

Minor: would be nice to substring match, but not important